app/controllers/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show; end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
```

Comment:
```suggestion
    @user = User.new user_params
```
bỏ ngoặc nhé, hạn chế dùng ngoặc